app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -173,13 +174,14 @@
     <icon drawable="@drawable/fairemail" package="eu.faircode.email" name="FairEmail" />
     <icon drawable="@drawable/fdroid" package="org.fdroid.fdroid" name="F-Droid" />
     <icon drawable="@drawable/files" package="com.android.documentsui" name="Files" />
+    <icon drawable="@drawable/files" package="com.android.filemanager" name="Files" />
     <icon drawable="@drawable/files" package="com.coloros.files" name="Files" />
     <icon drawable="@drawable/files" package="com.google.android.documentsui" name="Files" />
     <icon drawable="@drawable/files" package="com.huawei.hidisk" name="Files" />
     <icon drawable="@drawable/files" package="com.oneplus.filemanager" name="Files" />
     <icon drawable="@drawable/files" package="com.sec.android.app.myfiles" name="Files" />
     <icon drawable="@drawable/files" package="me.zhanghai.android.files" name="Files" />
-    <icon drawable="@drawable/files" package="com.android.filemanager" name="File Manager" />
```

Comment:
Seems like a mistake?

Reply:
I renamed that on purpose to avoid redundancy on the Lawnicons app itself